cmd/format-coverage.go

```diff
@@ -105,6 +105,10 @@ func (f CoverageFormatter) Save() error {
 		return errors.WithStack(err)
 	}
 
+	if len(rep.SourceFiles) == 0 {
+		return errors.WithStack(errors.New("Empty report. Could not find coverage info for source files"))
```

Comment:
💄 I think `Could not find coverage info for source files.` (note the `.` at the end) would suffice.